Example/ParleyExample/Controllers/IdentifierViewController.swift

```diff
@@ -154,24 +174,30 @@ class IdentifierViewController: UIViewController {
             startChatDemo()
         }
     }
-    
+
     // Start a chat based on the input
     private func startChatDemo() {
-        if let customerIdentification = self.customerIdentificationTextView.text?.trimmingCharacters(in: .whitespacesAndNewlines), !customerIdentification.isEmpty {
-            self.startChat(customerIdentification: customerIdentification)
-        } else if let secret = self.identifierTextView.text?.trimmingCharacters(in: .whitespacesAndNewlines), !secret.isEmpty, secret.count == 20 {
-            self.startChat(secret: secret)
+        if
+            let customerIdentification = customerIdentificationTextView.text?
+                .trimmingCharacters(in: .whitespacesAndNewlines), !customerIdentification.isEmpty
+        {
+            startChat(customerIdentification: customerIdentification)
+        } else if
+            let secret = identifierTextView.text?.trimmingCharacters(in: .whitespacesAndNewlines), !secret.isEmpty,
+            secret.count == 20
```

Comment:
This looks syntax-wise not very nice, not sure if moving the parts after the `,` to a newline would make the formatting nicer?

Reply:
I added some newlines for the mentioned code, so that the separate statements in the if-statement have an individual newline.
Swiftlint or swifformat is not doing this autmatically.
I can image that it is a trigger to make a extension on String with a var to trimWhitespacesAndNewlines to make the code shorter. And make a computed var on the view customerIdentificationIsNotEmpty. This would improve the code on[ Law of Demeter violations](https://daddycoding.com/2023/03/27/the-law-of-demeter/#:~:text=The%20Law%20of%20Demeter%20is,the%20objects%20that%20they%20manipulate.)

Do you see more places where you think the formatting is uglier than before?

Reply:
Thanks! Nope, rest looks good.

Agreed, an extension would make that part shorter, probably will do that sometime. For now, this looks fine 👍